### PR TITLE
feat: Add waf/wafv2 logging config

### LINKED ISSUE #360: Extend WAF / WAFv2 logging configuration struct
### Use Case

Currently the logging configuration attr is referring to the destination arn's only.
We should consider to extend the struct to the other logging configuration attrs.

### Additional context

Wafv2
https://github.com/aws/aws-sdk-go-v2/blob/cc93d47779759286d4b82c0be987afcb5bb0f1e7/service/wafv2/types/types.go#L1005

Waf
https://github.com/aws/aws-sdk-go-v2/blob/cc93d47779759286d4b82c0be987afcb5bb0f1e7/service/waf/types/types.go#L778

### PR DESCRIPTION
**Pull request by zagronitay**
🎉 Thank you for making CloudQuery awesome by submitting a PR 🎉

#### Summary

---

Use the following steps to ensure your PR is ready to be reviewed

- [x] Read the [contribution guidelines](../CONTRIBUTING.md) 🧑‍🎓
- [x] Run `go fmt` to format your code 🖊
- [x] Update or add tests. Learn more about testing [here](https://docs.cloudquery.io/docs/developers/sdk/testing) 🧪
- [x] Update the docs by running `go run ./docs/docs.go` and committing the changes 📃
- [x] If adding a new resource, add [relevant Terraform files](../terraform) in a separate PR 📂
- [x] Ensure the status checks below are successful ✅

### CODE REVIEW

**Review by roneli — APPROVED** [review]

**Comment by bbernays** [comment]
This will require a change to the AWS policy:
https://github.com/cloudquery-policies/aws/search?l=SQL&q=logging_configuration

**Review by bbernays — APPROVED** [review]
LGTM

**Review by bbernays — APPROVED** [review]

File: resources/services/waf/rules.go [hunk 1]
```diff
@@ -56,11 +55,10 @@ func WafRules() *schema.Table {
...
 			{
-				Name:          "aws_waf_rule_predicates",
-				Description:   "This is AWS WAF Classic documentation",
-				Resolver:      fetchWafRulePredicates,
-				Options:       schema.TableCreationOptions{PrimaryKeys: []string{"rule_cq_id", "data_id"}},
-				IgnoreInTests: true,
+				Name:        "aws_waf_rule_predicates",
+				Description: "This is AWS WAF Classic documentation",
+				Resolver:    fetchWafRulePredicates,
+				Options:     schema.TableCreationOptions{PrimaryKeys: []string{"rule_cq_id", "data_id"}},
```
Comment (yevgenypats) [inline resources/services/waf/rules.go:61]: I think this is unneeded anymore?

**Review by yevgenypats — CHANGES_REQUESTED** [review]
Looks good. I think child PKs are not needed anymore.

**Review by bbernays — APPROVED** [review]

**Pull request merged**